Modules/TRD/src/TrackletsTask.cxx

```diff
@@ -100,114 +94,182 @@ void TrackletsTask::drawTrdLayersGrid(TH2F* hist)
+      "Tracklet distribution over Halfchambers noise in;HalfChamber ID; Counts",
+      1080, 0, 1080);
   getObjectsManager()->startPublishing(mTrackletHCIDn);
-  mTrackletPositionn = new TH1F("trackletposnoise", "Uncalibrated Position of Tracklets noise in;Position;Counts", 1400, -70, 70);
+  mTrackletPositionn =
+      new TH1F("trackletposnoise",
+               "Uncalibrated Position of Tracklets noise in;Position;Counts",
+               1400, -70, 70);
   getObjectsManager()->startPublishing(mTrackletPositionn);
-  mTrackletPositionRawn = new TH1F("trackletposrawnoise", "Raw Position of Tracklets noise in;Position;Counts", 2048, 0, 2048);
+  mTrackletPositionRawn = new TH1F(
+      "trackletposrawnoise",
+      "Raw Position of Tracklets noise in;Position;Counts", 2048, 0, 2048);
   getObjectsManager()->startPublishing(mTrackletPositionRawn);
-  mTrackletsPerEventn = new TH1F("trackletspereventn", "Number of Tracklets per event noise in;Tracklets in Events;Counts", 25000, 0, 25000);
+  mTrackletsPerEventn = new TH1F(
+      "trackletspereventn",
+      "Number of Tracklets per event noise in;Tracklets in Events;Counts",
+      25000, 0, 25000);
   getObjectsManager()->startPublishing(mTrackletsPerEventn);
-  mTrackletsPerTimeFrame = new TH1F("trackletspertimeframe", "Number of Tracklets per timeframe;Tracklets in TimeFrame;Counts", 25000, 0, 500000);
+  mTrackletsPerTimeFrame = new TH1F(
+      "trackletspertimeframe",
+      "Number of Tracklets per timeframe;Tracklets in TimeFrame;Counts", 25000,
+      0, 500000);
   getObjectsManager()->startPublishing(mTrackletsPerTimeFrame);
-  mTrackletsPerTimeFrameCycled = new TH1F("trackletspertimeframecycled", "Number of Tracklets per timeframe, this cycle;Tracklets in TimeFrame;Counts", 25000, 0, 500000);
+  mTrackletsPerTimeFrameCycled =
+      new TH1F("trackletspertimeframecycled",
+               "Number of Tracklets per timeframe, this cycle;Tracklets in "
+               "TimeFrame;Counts",
+               25000, 0, 500000);
   getObjectsManager()->startPublishing(mTrackletsPerTimeFrameCycled);
-  mTriggersPerTimeFrame = new TH1F("triggerspertimeframe", "Number of Triggers per timeframe;Triggers in TimeFrame;Counts", 100, 0, 100);
+  mTriggersPerTimeFrame =
+      new TH1F("triggerspertimeframe",
+               "Number of Triggers per timeframe;Triggers in TimeFrame;Counts",
+               100, 0, 100);
   getObjectsManager()->startPublishing(mTriggersPerTimeFrame);
+  mEventsPerTimeFrame = new TH1F("eventswithdigitspertimeframe",
+                                 "Number of Events with Digits per Time "
+                                 "Frame;Event with digits in Time Frame;Counts",
+                                 100, 0, 100);
+  getObjectsManager()->startPublishing(mEventsPerTimeFrame);
+  mDigitsPerEvent = new TH1F(
+      "digitsperevent", "Number of Digits per event;Digits in Event;Counts",
+      100, 0, 50000);
+  getObjectsManager()->startPublishing(mDigitsPerEvent);
```

Comment:
Why is this in the TrackletsTask ? When it is solely dealing with digits?
Also in DigitTask is the same spectrum : https://github.com/AliceO2Group/QualityControl/blob/72c5618f40f3529a21b26b6751a32bc8cf3f1f44/Modules/TRD/src/DigitsTask.cxx#L214

Reply:
> Why is this in the TrackletsTask ? When it is solely dealing with digits? Also in DigitTask is the same spectrum :
> 
> https://github.com/AliceO2Group/QualityControl/blob/72c5618f40f3529a21b26b6751a32bc8cf3f1f44/Modules/TRD/src/DigitsTask.cxx#L214

Yes I checked the digit task now. This is already in the digit task. Will not include it.